deployment/cf-templates/identification.json

```diff
@@ -1960,6 +1966,105 @@
                 "LogGroupName" : { "Ref": "LogGroupLambdaEvaluateAMIPublicAccess" }
             }
         },
+        "LambdaInitiateECSExternalImageSourceEvaluation": {
```

Comment:
We have recently moved this staff to nested stack. Please use this as an example: https://github.com/dowjones/hammer/blob/master/deployment/cf-templates/identification.json#L822-L861
This helps to overcome 200 resources per stack limit and make our templates more organized and structures.